test/UnitTests/MSTest.Core.Unit.Tests/Attributes/DynamicDataAttributeTests.cs

```diff
@@ -293,7 +306,18 @@ public static IEnumerable<object[]> NullProperty
         }
 
         /// <summary>
-        /// Gets the reusable test data property.
+        /// Gets the empty test data property.
+        /// </summary>
+        public static IEnumerable<object[]> EmptyProperty
+        {
+            get
+            {
+                return new object[][] { };
+            }
+        }
+
+        /// <summary>
+        /// Gets the wrong test data property.
         /// </summary>
         public static IEnumerable<object[]> WrongDataTypeProperty
```

Comment:
Out of the scope of this PR maybe. Why do we WrongDataTypeProperty? How is that different from Null property. 
And why are these public properties and not private members ?

Reply:
I think the intent was different but it is not very clear. Correcting it.